repose-aggregator/components/filters/rate-limiting-filter/src/main/java/org/openrepose/filters/ratelimiting/RateLimitingServiceHelper.java

```diff
@@ -68,7 +69,7 @@ public MimeType queryCombinedLimits(HttpServletRequest request, MimeType preferr
     }
 
     public void trackLimits(HttpServletRequest request, int datastoreWarnLimit) throws OverLimitException {
-        service.trackLimits(getPreferredUser(request), getPreferredGroups(request), decodeURI(request.getRequestURI()), request.getParameterMap(), request.getMethod(), datastoreWarnLimit);
+        service.trackLimits(getPreferredUser(request), getPreferredGroups(request), decodeURI(request.getRequestURI()), HttpServletRequestWrapper$.MODULE$.parseQueryString(request.getQueryString()), request.getMethod(), datastoreWarnLimit);
```

Comment:
Actually, this call appears to work without the `$.MODULE$` madness. Can that be dropped?

This gives a brief explanation, and seems to state that it can be:
http://lampwww.epfl.ch/~michelou/scala/using-scala-from-java.html